src/quarterback/useDocStore.ts

```diff
@@ -46,6 +46,7 @@ export const useDocStore = create(
           manuscript_model_id: manuscriptID,
           project_model_id: projectID,
           doc: {},
+          user_model_id: ''
```

Comment:
@nghazzawi-atypon is that correct from our side? @hhusban , should that be auth based? Why would FE provide user id since we can operate only under a single user and can't really manage other users access.

Reply:
@mbartenev-atypon I will  hold on this until we get a response from @hhusban

Reply:
@mbartenev-atypon  no that shouldn't be right, I rolled back my changes